app/views/mainstream_browse_pages/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= content_for :page_title, "Mainstream browse page" %>
+
+<header class="heading-with-actions">
+  <h1>Mainstream browse page</h1>
+
+  <% if gds_editor? %>
```

Comment:
This conditional isn't necessary.  Mainstream browse pages are restricted wholesale in the controller.